types/filters/parse.go

```diff
@@ -221,6 +221,24 @@ func (filters Args) ExactMatch(field, source string) bool {
 	return false
 }
 
+// OnlyOneExactMatch returns true if there is only one filters and the source matches exactly this one.
+func (filters Args) OnlyOneExactMatch(field, source string) bool {
+	fieldValues, ok := filters.fields[field]
+	//do not filter if there is no filter set or cannot determine filter
+	if !ok || len(fieldValues) == 0 {
+		return true
```

Comment:
Perhaps I should have a look at how they're used exactly, because this feels very confusing.